packages/angular/cli/commands/update-impl.ts

```diff
@@ -178,17 +181,53 @@ export class UpdateCommand extends Command<UpdateCommandSchema> {
         return false;
       }
 
+      this.logger.info(colors.green(`${colors.symbols.check} Migration succeeded.`));
+
       // Commit migration
       if (commit) {
-        let message = `${packageName} migration - ${migration.name}`;
-        if (migration.description) {
-          message += '\n' + migration.description;
+        // Check if a commit is needed.
+        let commitNeeded: boolean;
```

Comment:
Can you extract this logic into a common function?  Looks like just the message is unique between the two instances?

Reply:
Done. I had to change some of the log text, but I think I was able to make it work reasonably well.